jisantuc/AFDG
# Let user switch two orthogonally adjacent tiles in the grid

Overview
-----

This PR makes it so that users can switch two tiles in the game grid,
provided the tiles border each other.

Notes
-----

Response to failed switch is currently not very ergonomic, captured in #42

Testing
-----

- add some stuff to some tiles
- click the switch tiles button
- click a tile you want to switch with a different one
- click another tile
- if your second tile borders your first one, you should observe the switch occur
- if not, nothing should happen and you'll have to try again

Closes #16
